crowdin/translated/api.nl.json

```diff
@@ -633,11 +633,11 @@
   },
   "math.isnan": {
     "message": "isnan",
-    "description": "(function name) Check is a value is not-a-number (NaN)."
+    "description": "(function name) Check if a value is not-a-number (NaN)."
   },
   "math.isnan.summary": {
-    "message": "Check is een waarde is geen getal (NaN).",
-    "description": "Check is a value is not-a-number (NaN)."
+    "message": "Check if is een waarde is geen getal (NaN).",
```

Comment:
Is "if is" correct?

Reply:
Hmm perhaps not. I'll take a look.

Reply:
https://crowdin.com/translate/microbitorg/6422/en-nl#329048

It was translated since the string update and reverse Google translates OK so I'm going with it.